src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/GameBeanValidationTest.java

```diff
@@ -21,27 +29,68 @@ void setup() {
     validatorFactory.close();
   }
 
-  @Test
-  void valid_player_is_accepted() {
+  @ParameterizedTest(name = "{0}")
+  @MethodSource("gameNames")
+  void validates_game_name(String name, ValidationListMatcher validationListMatcher) {
     Game game = new Game();
-    game.setName("test");
+    if (name != null) {
+      game.setName(name);
+    }
+
+    assertThat(validator.validate(game), validationListMatcher);
+  }
 
-    assertThat(validator.validate(game), empty());
+  private static Stream<Arguments> gameNames() {
+    return Stream.of(
+            Arguments.of("test", empty()),
+            Arguments.of(null, not(empty())),
```

Comment:
What does that check mean? If the gameName was null, then ...?

Reply:
the list of validation errors is not empty